smart-contracts/test/test_security.js

```diff
@@ -388,6 +528,11 @@ describe("Security Test", function () {
     });
 
     it("should allow users to unpeg troll token, but then does not receive", async function () {
+      // Add the token into white list
+      await state.bridgeBank.connect(operator)
```

Comment:
Let's add another test that updates the eth whitelist with a cosmos whitelisted token and then check to make sure that the error message is different than an already whitelisted ethereum asset.